test/unit/typing.test.js

```diff
@@ -880,6 +881,13 @@ suite('typing with auto-replaces', function() {
       assertLatex('\\sin\\pi');
     });
 
+    test('has lower "precedence" than operator names', function() {
+      mq.typedText('ppi');
+      assertLatex('\\operatorname{pp}i');
+      mq.keystroke('Left Left').typedText('i');
+      assertLatex('\\pi pi');
```

Comment:
because auto-conversion only happen when you type their last letter, nearby words aren't auto-converted. If you typed `i`, then moved the cursor in front and typed a `p`, you'd get `pi` not `π`, and nothing typed near it should auto-convert it (except backspacing and re-typing the `i`)